requirements.txt

```diff
@@ -1,2 +1,4 @@
 -e git+https://github.com/CartoDB/carto-python.git#egg=carto
 .
+google-cloud-bigquery
```

Comment:
I don't know the reason, but we are adding the dependencies here: https://github.com/CartoDB/cartoframes/blob/develop/setup.py#L21-L30 but I know requirements.txt is the usual one. How do you see it?

Reply:
Yes, we should use requirements.txt because of is the standard...

Reply:
But I will place dependencies in setup.py in the meantime